stdlib/sys.mlp

```diff
@@ -17,10 +18,10 @@
    your changes will be lost.
 *)
 
-type backend_type = 
+type backend_type =
   | Native
   | Bytecode
-  | Other of string 
+  | Other of string
```

Comment:
Removing whitespace adds noise to the PR. I usually do it as a big commit, using tools/check-typo to make sure I didn't miss any.

So: don't do it unless you're already changing the line for some other reason.